.github/workflows/CI.yaml

```diff
@@ -192,6 +193,9 @@ jobs:
           KEYCLOAK_UPLOADER_VALUE: "Dic1AEUG0nKnZ8me/3GMxgfyUmbwmJwDWNXkC7arLcij2BDB0xeOgk8ZpfZPKmFNikr9Is5I4+Nyk3MB9zs7mA=="
           KEYCLOAK_UPLOADER_SALT: "K9nzg086pCNjq8dxl84GZg=="
           KEYCLOAK_UPLOADER_PASSWORD: "password"
+          KEYCLOAK_REVIEWER_VALUE: "Dic1AEUG0nKnZ8me/3GMxgfyUmbwmJwDWNXkC7arLcij2BDB0xeOgk8ZpfZPKmFNikr9Is5I4+Nyk3MB9zs7mA=="
+          KEYCLOAK_REVIEWER_SALT: "K9nzg086pCNjq8dxl84GZg=="
+          KEYCLOAK_REVIEWER_PASSWORD: "password"
           KEYCLOAK_ADMIN: "admin"
           KEYCLOAK_ADMIN_PASSWORD: "password"
```

Comment:
Auch hier die Frage:
Ich konnte keinen Test finden, in dem man sich als ADMIN einloggt.  Theoretisch könnten wir daher zumindest das PASSWORD rausschmeißen oder?
Und das setup des ADMIN users in unserem Cypress Test-setup auch.
Was meint ihr?

Reply:
Don't we run the administration test and the register new user registration test in the CI? In those tests we log into the admin console of keycloak.

Reply:
1.) Pls fix this:

![image](https://github.com/d-fine/Dataland/assets/99255296/204673af-33d1-447f-acc1-7b678244751b)

Reply:
2.) Pls use `getStringCypressEnv()` for this, and also for all other cases where we get a cypress env and want to cast it as string:

![image](https://github.com/d-fine/Dataland/assets/99255296/3cd40819-3136-4665-b8b6-93e2074f171a)

Reply:
> Don't we run the administration test and the register new user registration test in the CI? In those tests we log into the admin console of keycloak.

Yes srry I referenced the wrong line.  I don't mean `KEYCLOAK_ADMIN_PASSWORD`.
I mean `KEYCLOAK_DATALAND_ADMIN_PASSWORD`

That one is used here to set dataland-admin login credentials:

![image](https://github.com/d-fine/Dataland/assets/99255296/f92a8390-3bac-48d7-bb85-306dcb529ec2)

But `admin_pw` and `admin_name` are never used in any test to log in.


Currently `reviewer_name` and `reviewer_pw` aren't used either, but they probably will when we write frontend tests for the reviewing process, right?


But `KEYCLOAK_DATALAND_ADMIN_PASSWORD` and `admin_pw` and `admin_name` are not being used and therefore unnecessary at the moment.

Reply:
Ok, yes true. There is no log in using data_admin, so these credentials should be removed.

Reply:
1.) addressed and to be pushed

Reply:
2.) addressend and pushed. Envs to be checked

Reply:
Ale users are being used now.